frontend/package.json

```
@@ -12,6 +12,7 @@
     "check": "prettier --check \"./src/**/*.{js,jsx,ts,tsx,css,scss}\""
   },
   "dependencies": {
+    "@bcgov/design-system-react-components": "^0.5.1",
```

```suggestion
    "@bcgov/design-system-react-components": "0.5.1",
```
Best to use exact versions to ensure compatibility